docs/TimerMocks.md

```diff
@@ -53,6 +53,8 @@ test('do something with real timers', () => {
 });
 ```
 
+Currently, two implementations of the fake timers are present - `modern` and `legacy`, where `legacy` is still the default one. You can read how to enable `modern` implementation [here](/blog/2020/05/05/jest-26#new-fake-timers).
```

Comment:
legacy is not the default as of Jest 27

Reply:
@SimenB in 27 version docs, should i add `modern` as the default one

Reply:
In 26 both versions exist. 26.x default is `legacy`, 27.x (and this one in `/docs`) `modern` is the default one

Reply:
@SimenB also for all the docs do i need to mention about the existence of both modern and legacy